core/ui/src/main/java/com/google/samples/apps/nowinandroid/core/ui/NewsResourceCardList.kt

```diff
@@ -59,10 +54,6 @@ fun LazyListScope.userNewsResourceCardItems(
                 analyticsHelper.logNewsResourceOpened(
                     newsResourceId = userNewsResource.id,
                 )
-                when (onItemClick) {
-                    null -> launchCustomChromeTab(context, resourceUrl, backgroundColor)
```

Comment:
You've removed the code which opens the news resource :) 

Please add back the line: 

```launchCustomChromeTab(context, resourceUrl, backgroundColor)```

Reply:
Sorry my bad, sending the patch. 
I had a question, is there a way I can run these "build checks" on my local machine?

Reply:
alright, thanks a ton